packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/tags/TagsDataPrep.java

```diff
@@ -60,4 +60,23 @@ public void init() throws Exception
             });
 
     }
+
```

Comment:
Can we remove REQUIRE_SOLR from this file?

Reply:
I've also removed the @Test annotation as there's no tests ran in the dataPrep class